samples/BenchmarkDotNet.Samples/IntroColdCpuCache.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Jobs;
+
+namespace BenchmarkDotNet.Samples
+{
+    [Config(typeof(CompereCacheClearingStrategies))]
+    public class IntroColdCpuCache
+    {
+        private class CompereCacheClearingStrategies : ManualConfig
+        {
+            public CompereCacheClearingStrategies()
+            {
+                Add(Job.Default.WithCacheClearingStrategy(CacheClearingStrategy.Allocations));
+                Add(Job.Default.WithCacheClearingStrategy(CacheClearingStrategy.None));
+                Add(Job.Default.WithAffinity((IntPtr) 1).WithCacheClearingStrategy(CacheClearingStrategy.Allocations));
+                Add(Job.Default.WithAffinity((IntPtr) 1).WithCacheClearingStrategy(CacheClearingStrategy.None));
+
+                Add(StatisticColumn.Q1,
```

Comment:
We don't actually need Q1/Median/Q3/Skewness/Kurtosis in this sample. Instead of this, we need StatisticalTestColumn here.

Reply:
I've added: `[StatisticalTestColumn(StatisticalTestKind.MannWhitney, ThresholdUnit.Microseconds, 0.2, true)]`. I'm not sure if it is good.